modules/living_spaces_group/living_spaces_group.module

```diff
@@ -16,11 +18,24 @@ use Drupal\Core\Url;
 use Drupal\group\Entity\GroupInterface;
 use Drupal\group\Entity\GroupRoleInterface;
 use Drupal\group\Entity\GroupContentInterface;
-use Drupal\profile\Entity\ProfileInterface;
 use Drupal\user\UserInterface;
 use Drupal\user\Entity\Role;
 use Drupal\Core\Cache\RefinableCacheableDependencyInterface;
 
+/**
+ * Implements hook_views_query_alter().
+ */
+function living_spaces_group_views_query_alter(ViewExecutable $view, QueryPluginBase $query) {
+  if ('users' == $view->id() &&
+    'employee' == $view->current_display &&
+    \Drupal::currentUser()->hasPermission('administer users')
+  ) {
+    // Skip contextual filter by space ids for admins.
+    $where = &$query->getWhere();
+    $where[0]['conditions'] = [];
+  }
+}
```

Comment:
It would be better to not alter the query, but to control initial filter since we can do that.
\Drupal\living_spaces_group\Plugin\views\argument_default\LivingSpacesGroupJoinedSpaces::getArgument() can return 'all' if user has 'administer users' access